executor/set_test.go

```diff
@@ -15,6 +15,7 @@ package executor_test
 
 import (
 	"context"
+	"github.com/pingcap/tidb/config"
```

Comment:
Keep the sys libs seperate with the 3rd libs. Please move it down to line 20.

Reply:
i  already finished it